sdks/uniswapx-sdk/src/constants.ts

```diff
@@ -81,6 +84,11 @@ export const REACTOR_ADDRESS_MAPPING: ReactorMapping = {
     [OrderType.Dutch]: "0xD6c073F2A3b676B8f9002b276B618e0d8bA84Fad",
     [OrderType.Relay]: "0x0000000000A4e21E2597DCac987455c48b12edBF",
   },
+  42161: {
```

Comment:
We need to add this to `sdks/uniswapx-sdk/src/constants.test.ts` as well, right?

Reply:
oh true